core/task/manager.go

```diff
@@ -98,28 +102,92 @@ func (m*Manager) NewTaskForMesosOffer(offer *mesos.Offer, descriptor *Descriptor
 	return
 }
 
-func (m *Manager) RefreshClasses() (err error) {
+// TODO: Any reason to be a *Manager method? If not then should it reside here?
```

Comment:
It doesn't have to be in `task.Manager`, it can also be in `repo.Manager` if we can make it independent from the `task` package.

Reply:
Can't be independent from the `task` package and thus creates an import cycle. Will leave it here.